tests/test_nightly.py

```
@@ -298,5 +515,187 @@ def test_no_api_key_exits(
+            for i in range(3)
+        ]
+        placements = plan_nightly(
+            tasks,
+            today=self.today,
+            capacity_hours=50.0,
+            default_task_hours=1.0,
+        )
+        self.assertEqual(len(placements), 3)
+        # All three default to 1hr; week capacity is 50 — they all
+        # fit in the week containing today (Sunday 2026-05-17, i.e.
+        # Mon 5/11 – Sun 5/17). week2_start is Mon of the next week.
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)